Source/JavaScriptCore/llint/LLIntSlowPaths.cpp

```diff
@@ -863,6 +863,13 @@ static JSValue performLLIntGetByID(BytecodeIndex bytecodeIndex, CodeBlock* codeB
 
             if (!(--metadata.hitCountForLLIntCaching))
                 setupGetByIdPrototypeCache(globalObject, vm, codeBlock, bytecodeIndex, metadata, baseCell, slot, ident);
+        } else if (hasAlwaysSlowPutContiguous(baseCell->indexingMode()) && ident == vm.propertyNames->length) {
```

Comment:
You want Options::useLLIntICs here, too

Reply:
Please note that a parent branch is guarded by a `Options::useLLIntICs` on L816.